src/main/java/gregtech/api/metatileentity/implementations/MTEHatch.java

```diff
@@ -40,7 +40,7 @@ public MTEHatch(String aName, int aTier, int aInvSlotCount, String[] aDescriptio
     }
 
     public static int getSlots(int aTier) {
-        return aTier < 1 ? 1 : aTier == 1 ? 4 : aTier == 2 ? 9 : 16;
+        return (aTier + 1) * (aTier + 1);
     }
 
```

Comment:
for more cleaner calculations i'd do the following changes to reduce the amount of repeated calculations and magic numbers and use them in other classes

```suggestion

    private int getOffsetTier() {
        return Math.max(0, mTier - 3);
    }
    private int getOffsetX() {
        return getOffsetTier() * 2;
    }
    private int getOffsetY() {
        return getOffsetTier() * 16;
    }
```